dashboard/app/controllers/admin_search_controller.rb

```diff
@@ -15,6 +15,9 @@ def find_students
     users = User.with_deleted
 
     # If requested, filter...
+    if params[:usernameFilter].present?
+      users = users.where("username LIKE ?", "%#{params[:usernameFilter]}%")
```

Comment:
Please use `sanitize_sql_like` to filter potential SQL injections in the params:
```suggestion
      users = users.where("username LIKE ?", "%#{sanitize_sql_like(params[:usernameFilter])}%")
```

I would also recommend using ActiveRecord Arels:
```suggestion
      users = users.where(User.arel_table[:username].matches("%#{params[:usernameFilter]}%"))
```

Reply:
Note: Leading wildcards (e.g., '%TERM') prevent the use of the field index for this query.